examples/backbone_marionette/js/TodoMVC.Application.js

```diff
@@ -0,0 +1,27 @@
+/*global Backbone, TodoMVC:true */
+
+var TodoMVC = TodoMVC || {};
+
+(function () {
+	'use strict';
+
+	var TodoApp = Backbone.Marionette.Application.extend({
```

Comment:
(we are only using start but triggerMethod on Object would work just as well right¿)